tests/conftest.py

```diff
@@ -201,6 +201,13 @@ def run_function_from_file(item, params=None):
 
     # Override environment variables for the subprocess
     env = os.environ.copy()
+    env.update(
+        {
+            # Add the tests path to the Python path so tests can import helpers
+            # eg. from tests.webclient import PingFilter
+            "PYTHONPATH": os.path.dirname(os.path.dirname(ddtrace.__file__)),
```

Comment:
we might want to join to any existing `PYTHONPATH` entries, in case it is passed as a parameter by the marker

Reply:
I made sure that the user-specified env would overwrite this entry, but yeah merging might be more intuitive. I wasn't sure if it would be or not if there would be too much magic.

Is this a blocker @P403n1x87 or are you good with the change as-is and we can improve in the future?

Reply:
I thought we had some tests that set `PYTHONPATH` but either I misremembered or it has been changed. I think we're probably fine for now, but IMO we should make sure we can modify the `PYTHONPATH` from tests by prepending the tests folder to any existing value of the variable.